### ISSUE TITLE
[BUG] Turning off "Emulate old character sounds" isn't working

### ISSUE THREAD

**Issue by TimMarvelHero** [pizzaboxer/bloxstrap#2792]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### What problem did you encounter?

So basically I turned on the old character sounds for a test and decided to turn it off afterwards, it's turned off in Bloxstrap's settings but whenever I join a game it still has the old sounds, I even reinstalled Roblox and Bloxstrap and it still repeated this issue.

### Bloxstrap Log

N/A

**Comment by Storm-930** [comment]
do the following:
- read the preliminary instructions, to be certain that your problem has not already been addressed.
- thoroughly look through the available Wiki articles to try and find a solution
- use bloxstrap 2.7.0
- not mark the fourth as it shows that you did none of the above
_seriously, these people are really starting to piss me off_

**Comment by livenolife** [comment]
we believe you

**Comment by TimMarvelHero** [comment]
@Storm-930 I am exactly on version 2.7.0

**Comment by TimMarvelHero** [comment]
@Storm-930 I am literally on version 2.7.0 and it has this problem

**Comment by bluepilledgreat** [comment]
1. Make sure the option is definitely off
2. Open the menu, go to behaviour and enable 'Force Roblox reinstallation'

**Comment by TimMarvelHero** [comment]
@bluepilledgreat it has worked thank you so much!
